pkg/cmd/step/restore/step_restore_from_backup.go

```diff
@@ -93,7 +93,7 @@ func (o *FromBackupOptions) Run() error {
 	// check if a velero schedule exists
 	scheduleExists, err := velero.DoesVeleroBackupScheduleExist(apiClient, o.Namespace)
 	if err != nil {
-		errors.Wrap(err, "when trying to check for velero schedules")
+		return errors.Wrap(err, "when trying to check for velero schedules")
```

Comment:
lol, apparently that happened quite often in the codebase. Wrapping the error, but not returning it. Definitely shaking out some bugs here 👍